dao.py

```diff
@@ -619,6 +618,25 @@ def get_user_by_session_id_or_none(self, session_id):
     def get_user_by_region(self, regions):
         pass
 
+    def get_is_superadmin(self, user_id):
+        user = None
+        if self.users_col.find_one({'_id': user_id}):
+            user = M.User.load(self.users_col.find_one({'_id': user_id}))
+            return user.admin_level == 'SUPER'
+        else:
+            return False
+
+    def set_user_admin_level(self, user_id, admin_level):
```

Comment:
do we actually use this function anywhere else in the code yet? if not, I think it would be fine to just replace it with a simple `update_user` function, in the same style as `update_player` and `update_region`.